spec/defines/windows_firewall/exception_spec.rb

```diff
@@ -387,16 +387,16 @@
       let(:title) { 'Windows Remote Management' }
       let :params do
         {
-          ensure: 'present', direction: 'fubar', action: 'allow', enabled: 'yes',
-          protocol: 'TCP', local_port: '5985',
+          ensure: 'present', direction: 'fubar', action: 'allow', enabled: true,
+          protocol: 'TCP', local_port: 5985,
           display_name: 'Windows Remote Management', description: 'Inbound rule for WinRM'
         }
       end
 
       it do
         expect do
           is_expected.to contain_exec('set rule Windows Remote Management')
-        end.not_to raise_error
+        end.to raise_error(Puppet::Error, %r{fubar})
```

Comment:
The error message regex could probably be a little longer than just 'fubar'?

Reply:
@alexjfisher, I'm afraid that's what Puppet printed as error in my test case. I'd be happy if you could test and enhance it! Maybe I had some error somewhere....

Reply:
lulz! I'll take a look at that.